### ISSUE TITLE
With CFT tools 1+ and TF v0.13+, CI experiencing terraform-provider-kubernetes#1028

### ISSUE THREAD

**Issue by apeabody** [terraform-google-modules/terraform-google-gke-gitlab#89]
### TL;DR

https://github.com/hashicorp/terraform-provider-kubernetes/issues/1028

### Observed behavior

       Error: Get "http://localhost/apis/storage.k8s.io/v1/storageclasses/pd-ssd": dial tcp 127.0.0.1:80: connect: connection refused
       
         with module.gke-gitlab.module.gke-gitlab.kubernetes_storage_class.pd-ssd,
         on ../../../main.tf line 284, in resource "kubernetes_storage_class" "pd-ssd":
        284: resource "kubernetes_storage_class" "pd-ssd" {
       
       
       Error: Get "http://localhost/api/v1/namespaces/default/secrets/gitlab-pg": dial tcp 127.0.0.1:80: connect: connection refused
       
         with module.gke-gitlab.module.gke-gitlab.kubernetes_secret.gitlab_pg,
         on ../../../main.tf line 298, in resource "kubernetes_secret" "gitlab_pg":
        298: resource "kubernetes_secret" "gitlab_pg" {
       
       
       Error: Get "http://localhost/api/v1/namespaces/default/secrets/gitlab-rails-storage": dial tcp 127.0.0.1:80: connect: connection refused
       
         with module.gke-gitlab.module.gke-gitlab.kubernetes_secret.gitlab_rails_storage,
         on ../../../main.tf line 310, in resource "kubernetes_secret" "gitlab_rails_storage":
        310: resource "kubernetes_secret" "gitlab_rails_storage" {
       
       
       Error: Get "http://localhost/api/v1/namespaces/default/secrets/gitlab-registry-storage": dial tcp 127.0.0.1:80: connect: connection refused
       
         with module.gke-gitlab.module.gke-gitlab.kubernetes_secret.gitlab_registry_storage,
         on ../../../main.tf line 327, in resource "kubernetes_secret" "gitlab_registry_storage":
        327: resource "kubernetes_secret" "gitlab_registry_storage" {
       
       
       Error: Get "http://localhost/api/v1/namespaces/default/secrets/google-application-credentials": dial tcp 127.0.0.1:80: connect: connection refused
       
         with module.gke-gitlab.module.gke-gitlab.kubernetes_secret.gitlab_gcs_credentials,
         on ../../../main.tf line 347, in resource "kubernetes_secret" "gitlab_gcs_credentials":
        347: resource "kubernetes_secret" "gitlab_gcs_credentials" {

### Terraform Configuration

```hcl
N/A
```

### Terraform Version

```sh
N/A
```

**Comment by apeabody** [comment]
CI fixed with #91
